example01/package.json

```diff
@@ -1,6 +1,12 @@
 {
   "name": "example01",
   "private": true,
+  "scripts": {
+    "start": "npm run build",
+    "poststart": "./node_modules/serve/bin/serve.js .",
```

Comment:
It should be sufficient to just specify `serve .` here, as per https://docs.npmjs.com/misc/scripts#path.